Makefile

```diff
@@ -12,77 +12,45 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+CMDS=nfs-provisioner
+all: build
+
+include release-tools/build.make
+
 ifeq ($(REGISTRY),)
 	REGISTRY = quay.io/external_storage/
 endif
+
 ifeq ($(VERSION),)
 	VERSION = latest
 endif
-IMAGE = $(REGISTRY)nfs-ganesha-server-and-provisioner:$(VERSION)
-IMAGE_ARM = $(REGISTRY)nfs-ganesha-server-and-provisioner-arm:$(VERSION)
-MUTABLE_IMAGE = $(REGISTRY)nfs-ganesha-server-and-provisioner:latest
-MUTABLE_IMAGE_ARM = $(REGISTRY)nfs-ganesha-server-and-provisioner-arm:latest
-
 
-all build: build-x86_64
-.PHONY: all build
-
-build-x86_64:
-	GOOS=linux go build ./cmd/nfs-provisioner
-.PHONY: build-x86_64
-
-build-docker build-docker-x86_64:
-	GOOS=linux go build -o deploy/docker/x86_64/nfs-provisioner ./cmd/nfs-provisioner
-.PHONY: build-docker build-docker-x86_64
+IMAGE_ARM = $(REGISTRY)nfs-provisioner-arm:$(VERSION)
```

Comment:
cloud build and release tools already supports multiarch builds. I'm not sure we need these arm specific params and commands?

Reply:
The Dockerfiles for arm and amd64 are a bit different at the moment. We need to work on refactoring them to multi-arch compatible Docker files. Prior to refactoring, we need to add fix the e2e tests. 

The current users are building with the existing Docker files and make commands in forked repos - so tried to maintain that backward compatibility for now till the refactoring is completed.